chart/compass/templates/tests/ord-aggregator/ord-aggregator-test.yaml

```diff
@@ -22,14 +22,16 @@ spec:
     spec:
       serviceAccountName: {{ $.Chart.Name }}-e2e-ord-aggregator
       containers:
-        - name: ord-aggregator-tests
+        - name: {{ .Values.global.ordAggregator.containerName }}
```

Comment:
Going from `ord-aggregator-tests` to the parametarized `{{ .Values.global.ordAggregator.containerName }}` breaks the sandbox tests (as we noticed yesterday).

Make this `name` property value `{{ .Values.global.ordAggregator.containerName }}-tests` instead so that the sandbox make targets keep working